lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloDriver.java

```diff
@@ -68,6 +68,21 @@ public void onNext(final StartTime startTime) {
   }
 
   /**
+   * Handles the StartTime event: Request as single Evaluator.
+   */
+  public final class RestartHandler implements EventHandler<StartTime> {
```

Comment:
Can we move this to a different example? E.g. HelloWithRestart or such? I fear that its presence will confuse the audience of the `HelloREEF` example, newbies to REEF.